clients/fides-js/src/docs/fides-options.ts

```diff
@@ -127,4 +127,12 @@ export interface FidesOptions {
    * Defaults to `true`.
    */
   fides_tcf_gdpr_applies: boolean;
+
+  /**
+   * When `true`, shows fides.js overlay UI on load. This deletes the fides_consent cookie as if no preferences have been saved on reload.
```

Comment:
nit: This isn't actually accurate though, right? This won't force anything to be shown - it only clears the cookie. There will still be no UI shown if the location doesn't match, `fides_disable_banner` is true, etc.

Anyways, I updated the language here: https://github.com/ethyca/fides/pull/4816/files#diff-f3ea8ea74e0a542fd60a2896722c8e25e63153ebccafacbdc6ae0484d3d95b8aR57